File: Python Code/sum_circles.py
#sum_circles.py
#
#	Finds the sum of all the areas of the circles and adds them together
#	This code uses variable length parameters. Good ref.
#

import math
def sum_circles(*args):

	result = 0.0
	i = 0

	for i in args:
		result = result + getArea(i)

	result = int(round(result))

	return "We have this much circle: " + str(result)
    
def getArea(diameter):
	return math.pi*pow((diameter/2.0), 2)

File: JavaScript/CodeWars Katas/sumCircles.js
/* sumCircles.js
*
*	Check sum_circles.py for a description
*/
function sumCircles() 
{

	var result = 0;
  
	for(var i = 0; i < arguments.length; ++i)
	{
		// Area of circle
		var leArea = Math.PI*Math.pow((arguments[i]/2), 2);
		result = result + leArea;
	}
  // Round of the area
  result = Math.round(result);
  
	return ("We have this much circle: " + result);

}